trusted_applet/net.go

```diff
@@ -452,5 +358,20 @@ func startNetworking() (err error) {
 	iface.EnableICMP()
 	iface.Link.AddNotify(&txNotification{})
 
+	// hook interface into Go runtime
+	net.SocketFunc = iface.Socket
+
+	http.DefaultClient = &http.Client{
+		Transport: &http.Transport{
+			ForceAttemptHTTP2:     true,
+			MaxIdleConns:          100,
+			IdleConnTimeout:       90 * time.Second,
+			TLSHandshakeTimeout:   5 * time.Second,
+			ExpectContinueTimeout: 1 * time.Second,
+			TLSClientConfig:       &tls.Config{InsecureSkipVerify: true},
```

Comment:
Do we want a TODO to add a root set to this before shipping?

Reply:
I have a follow-up PR waiting which `embeds`s the build-host's bundle, I can add a TODO to that to come up with a more reproducible way of doing that?

Reply:
(added one here too)